### ISSUE TITLE
Pro Pinball - Timeshock! (1997)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#2297]
Discord user Xavier_Velo has noted that in Timeshock! the resolution can be set higher if video memory is increased to 8mb. This needs Daum or X to accomplish. Using Daum set vmemsize=8 under [dosbox] & output=direct3d. Screen flipping must be disabled to allow max colours and res at the same time. Not sure about doing the increase though as 1600x1200 is bigger than a 1080 monitor resolution and even 1280x1024 is bigger than the window resolution we set dosbox to.

In the games audio menu you can also go into a configuration sub-menu and increase a number of audio quality options.

It has also been pointed out that there is a hissing\screeching sound during table loading in ECE, but Daum fixes this so even if we dont switch for res we should still do so for this sound issue

**Comment by Python-Exoproject** [comment]
Xavier_Velo has also mentioned he has some frame stuttering at the current cycles. While I have not experienced this I am noting it here and suggest that if there are no ill side effects we could raise the cycles some to correct it for anyone suffering his issue?

**Comment by Python-Exoproject** [comment]
SmilingSpectre has pointed out that Daum has an issue with both shift keys being pressed at the same time which is a major issue in a pinball game. Thankfully X works as well as Daum does and fixes\allows all points above with the obvious changes of output=openglnb and fullresolution=desktop being X.

**Comment by exoscoriae** [comment]
ok. I changed to X
increased vmemsize to 8
I *did not* change the resolution. However, how someone *can* change it if they choose.
I saw no cycle stuttering at all. Since I don't have the issue, I don't know how many cycles to raise the machine to help it. And frankly, if 120k cycles is causing stuttering for someone, that likely means their CPU is maxing out. So raising cycles may not help much in that case.
